etcdctl/ctlv3/command/printer_json.go

```diff
@@ -43,3 +58,46 @@ func printJSON(v interface{}) {
 	}
 	fmt.Println(string(b))
 }
+
+func printMemberListWithHexJSON(r clientv3.MemberListResponse) {
+	var buffer bytes.Buffer
+	var b []byte
+	buffer.WriteString("{\"header\":{\"cluster_id\":\"")
+	b = strconv.AppendUint(nil, r.Header.ClusterId, 16)
```

Comment:
@tangcong 

can you just redefine ClusterID and other IDs as a JSONHexID type? And define a json marshaller on it? So we do not need to reconstruct a full json encoder for the entire memberlistrepons.

Reply:
something like https://gist.github.com/nikravi/e4812ed46ad77d8d2aee60faddd205df#file-hexbytearray-go-L17-L37 for the ID type.

Reply:
thanks.i tried this implementation but memberlist response is a pb srtuct and we have to redfine a repeat struct, it looks not good. do you prefer this implementation?